fix: do not cache module resolution during launchpad dependency detection (#26726)
cypress-io/cypress: cli/CHANGELOG.md

<!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->

> ## 12.13.0

Reviewer: Should this be a patch version bump?

Author: Right, yes. I'll change this.

_Released 05/23/2023 (PENDING)_

**Bugfixes:**

> - Fixed an issue where newly installed dependencies are not correct detected during Component Testing setup. Addresses [#26685](https://github.com/cypress-io/cypress/issues/26685).

Reviewer: ```suggestion
- Fixed an issue where newly-installed dependencies would not be detected during Component Testing setup. Addresses [#26685](https://github.com/cypress-io/cypress/issues/26685).
```
